requirements.txt

```diff
@@ -1,2 +1,2 @@
-statsd>=3.2.1, <=3.2.2
-datadog>=0.17.0, <=0.20.0
+statsd==3.2.2
+datadog==0.28.0
```

Comment:
I would be careful of pinning in a library as you could create conflicts in the application. Instead I would pin/freeze all deps in the application.